runtime/bindings/python/src/openvino/opset8/ops.py

```diff
@@ -414,3 +414,108 @@ def gather_nd(
     }
 
     return _get_node_factory_opset8().create("GatherND", inputs, attributes)
+
+
+@nameable_op
+def prior_box(
```

Comment:
We also need to add this op to legacy python API

Reply:
added in abb47a7